app.js

```diff
@@ -318,18 +320,54 @@ function CookieStore(location, minCust, maxCust, avgCookie) {
   allStores.push(this);
 }
 
-CookieStore.prototype.randomNumber = function () {
-  return Math.floor(Math.random() * (this.maxCust - this.minCust) + this.minCust);
-};
-
-CookieStore.prototype.populateHourlySales = function () {
-  for (var i = 0; i < hours.length; i++) {
-    var customersPerHour = Math.round(this.randomNumber() * this.avgCookie);
-    this.hourlySales.push(customersPerHour);
+//render method
+Store.prototype.renderRow = function(){
+  var trElement = document.createElement('tr');
+  var tdElement = document.createElement('td');
+  tdElement.textContent = this.location;
+
+  trElement.appendChild(tdElement);
+  for(var i = 0; i < hours.length; i++){
+    tdElement.textContent = this.hourlySales[i];
+    trElement.appendChild(tdElement);
+    storeTable.appendChild(trElement);
+
+    tdElement = document.createElement('td');
+    tdElement.textContent = this.dailySales;
+    trElement.appendChild(tdElement);
   }
-};
+  Store.prototype.randomNumber = function () {
+    return Math.floor(Math.random() * (this.maxCust - this.minCust) + this.minCust);
+  };
+
+  Store.prototype.populateHourlySales = function () {
+    for (var i = 0; i < hours.length; i++) {
+      var customersPerHour = Math.round(this.randomNumber() * this.avgCookie);
+      this.hourlySales.push(customersPerHour);
+
+      Store.prototype.totalSold = function(){
+        var cookiesSoldCalc = 0;
+        for (var i = 0; i < hours.length; i++){
+          cookiesSoldCalc += this.hourlySales[i];
+        }
+        this.totalSold = cookiesSoldCalc;
+      };
+
+    }
+  };
 
-var pike = new CookieStore('1st and Pike', 23, 65, 6.3);
+  var firstPike = new Store('1st and Pike', 23, 65, 6.3);
+  // var seaTac = new Store('Seatac Airport', 3 ,24, 1.2);
+  // var seaCenter = new Store('Seattle Center',11,38,3.7);
+  // var capHill = new Store('Capitol Hill', 20, 38, 2.3);
+  // var alkiBeach = new Store('Alki Beach', 2,16, 4.6);
 
+  firstPike.randomNumber();
+  firstPike.cookieSales();
```

Comment:
Make sure you're calling the methods you actually defined, like `populateHourlySales`.